keras_cv/layers/preprocessing/with_bounding_boxes_test.py

```diff
@@ -71,24 +71,20 @@
 
 
 class WithLabelsTest(tf.test.TestCase, parameterized.TestCase):
-    @parameterized.named_parameters(
-        *TEST_CONFIGURATIONS,
-        ("CutMix", preprocessing.CutMix, {}),
-    )
-    def test_can_run_with_labels(self, layer_cls, init_args):
+    @parameterized.named_parameters(*TEST_CONFIGURATIONS)
+    def test_can_run_with_boudning_boxes(self, layer_cls, init_args):
         layer = layer_cls(**init_args)
 
         img = tf.random.uniform(
             shape=(3, 512, 512, 3), minval=0, maxval=1, dtype=tf.float32
         )
-        labels = tf.ones((3,), dtype=tf.float32)
+        bounding_boxes = tf.random.uniform((3, 2, 4), 0, 255, dtype=tf.float32)
```

Comment:
bounding_boxes should be of shape (3, 2, 5), not (3, 2, 4).

class labels are important!